website/docs/cli/code/index.mdx

```diff
@@ -12,33 +12,32 @@ user interface, and all of Terraform's workflows rely on configurations written
-potentially save you time and effort.
+to work with. The following commands can be integrated into your editing 
+workflow and may save you time and effort:
 
-- [The `terraform console` command](/terraform/cli/commands/console) starts an
+- [`terraform console`](/terraform/cli/commands/console): Start an 
   interactive shell for evaluating Terraform
-  [expressions](/terraform/language/expressions), which can be a faster way
-  to verify that a particular resource argument results in the value you expect.
+  [expressions](/terraform/language/expressions) to quickly verify that a 
+  particular resource argument results in the value you expect.
 
-- [The `terraform fmt` command](/terraform/cli/commands/fmt) rewrites Terraform
+- [`terraform fmt`](/terraform/cli/commands/fmt): Rewrite Terraform
   configuration files to a canonical format and style, so you don't have to
-  waste time making minor adjustments for readability and consistency. It works
+  waste time making minor adjustments for readability and consistency. `fmt` works
   well as a pre-commit hook in your version control system.
 
-- [The `terraform validate` command](/terraform/cli/commands/validate) validates the
+- [`terraform validate`](/terraform/cli/commands/validate): Validate the
   syntax and arguments of the Terraform configuration files in a directory,
   including argument and attribute names and types for resources and modules.
   The `plan` and `apply` commands automatically validate a configuration before
-  performing any other work, so `validate` isn't a crucial part of the core
-  workflow, but it can be very useful as a pre-commit hook or as part of a
+  performing any other work. `validate` isn't a crucial part of the core
+  workflow but can be very useful as a pre-commit hook or as part of a
   continuous integration pipeline.
 
-- [The `0.13upgrade` command](/terraform/cli/commands/0.13upgrade) and
-  [the `0.12upgrade` command](/terraform/cli/commands/0.12upgrade) can automatically
-  modify the configuration files in a Terraform module to help deal with major
+- [`0.13upgrade`](/terraform/cli/commands/0.13upgrade) and
+  [`0.12upgrade`](/terraform/cli/commands/0.12upgrade): Modify the configuration files in a Terraform module automatically to help deal with major
   syntax changes that occurred in the 0.13 and 0.12 releases of Terraform. Both
-  of these commands are only available in the Terraform version they are
-  associated with, and you are expected to upgrade older code to be compatible
+  commands are only available in the Terraform version they are
+  associated with. Make sure to upgrade older code to be compatible
   with 0.12 before attempting to make it compatible with 0.13. For more detailed
   information about updating code for new Terraform versions, see the [upgrade
-  guides](/terraform/language/upgrade-guides) in the Terraform language docs.
+  guides](/terraform/language/upgrade-guides).
```

Comment:
IDK the pattern but I would leave it out. If the user is interested and clicks the links they land on the page in the language docs anyway.

Just in case, you wanna revert my change: 

```suggestion
  guides](/terraform/language/upgrade-guides) in the Terraform language docs.
```